tests/unit/-private/properties/blurrable-test.js

```diff
@@ -204,7 +204,7 @@ moduleForProperty('blurrable', function(test) {
       <iframe></iframe>
       <select></select>
       <button></button>
-      <div contenteditable=true></div>
+      <div contenteditable></div>
```

Comment:
Removed the `true` here, because the code only cares about `contenteditable="false"`, so the `true` is not needed.